.travis.yml

```diff
@@ -1,14 +1,19 @@
-language: python
+anguage: python
 python:
   - "2.7"
 env:
   - DJANGO_SETTINGS_MODULE=settings
 
 before_install:
+  - export PATH=./node_modules/.bin:$PATH
+  - sudo apt-get update -qq
+  - sudo apt-get install -y npm
```

Comment:
not sure, the analytics-dashboard is using these commands to first install npm so I have included these, not sure if node will be available by default, I guess I can try be commenting these and then see if travis fails or not